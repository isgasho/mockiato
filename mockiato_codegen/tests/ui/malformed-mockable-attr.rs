#![feature(plugin)]
#![plugin(mockiato_codegen)]

#[mockable("Bar")]
trait Foo {}
