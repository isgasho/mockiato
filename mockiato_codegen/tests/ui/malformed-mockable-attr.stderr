error: Unsupported syntax for #[mockable]
 --> $DIR/malformed-mockable-attr.rs:4:12
  |
4 | #[mockable("Bar")]
  |            ^^^^^
  |
  = help: Example usage: #[mockable(name = "FooMock", derive(Debug))]

error: aborting due to previous error

